python/langsmith/client.py

```diff
@@ -3250,6 +3252,23 @@ def _tracing_thread_handle_batch(
 _AUTO_SCALE_DOWN_NEMPTY_TRIGGER = 4
 
 
+def _get_batch_ingest_config(client: Client) -> Dict[str, Any]:
+    try:
+        info = client.info
```

Comment:
so this will still fail on old on-prem isntances right? where we 404-> empty info dict -> uses default config? can we just fallback to not batching in this case?